src/app/wizard/step/cluster/component.ts

```diff
@@ -247,6 +257,14 @@ export class ClusterStepComponent extends StepBase implements OnInit, ControlVal
     return AdmissionPluginUtils.getPluginName(name);
   }
 
+  updateCNIPluginOptions() {
+    if (this.controlValue(Controls.CNIPlugin) === 'cilium') {
+      this.availableProxyModes = ['ipvs', 'iptables', 'eBPF'];
```

Comment:
hmm, I put lowercase `ebpf` here: https://github.com/kubermatic/kubermatic/pull/7861

Reply:
I will make it lowercase too